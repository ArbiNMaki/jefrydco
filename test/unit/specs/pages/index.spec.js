import { shallowMount } from "@vue/test-utils";

import Index from "~/pages/index";

describe("Index", () => {
  it("renders to match snapshot", () => {
    const wrapper = shallowMount(Index);
    expect(wrapper.html()).toMatchSnapshot();
  });
});
