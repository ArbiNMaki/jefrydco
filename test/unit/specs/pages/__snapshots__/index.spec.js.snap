// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Index renders to match snapshot 1`] = `
<div id="home" class="flex flex-wrap items-center justify-center h-screen">
  <div class="flex-initial">
    <app-profile-stub></app-profile-stub>
  </div>
</div>
`;
